lib/main.dart

```diff
@@ -93,8 +93,10 @@ class _MyHomePageState extends State<MyHomePage> {
 class MyRecipesScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    final recipeBloc = context.bloc<RecipePaginationBloc>();
```

Comment:
The recipeBloc needs to have the right accountID. 

I would use this bloc to call something like the following:
`recipeBloc.add(ReloadPages())`